### ISSUE TITLE
Redirect Request: Pension Management Centers

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#88766]
### Status
**[2024-07-25] [Jill]** information provided, unblocked
**[2024-07-25] [Fran]** Awaiting information on the new URL to redirect to.

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

| POC | Github alias(es) |
--- | --- 
| OCTO team point(s) of contact: | @RLHecht  |
| Content point of contact: |  |
| IA point of contact: | @jennymayoco  |

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you need Public Websites to implement this redirect?
- [x] Yes
- [ ] No - Please note: Redirect still needs approval from IA and PW will do PR review

### Implementation date
When does this request need to be live? **NOTE:** Redirects deploy M-Th. Friday launches should be avoided.
Choose one:
- [ ] On a specific date, connected to a program or publishing requirement. **DATE:**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [x] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/pension/pension-management-centers/ | https://www.va.gov/pension/how-to-apply/

7/31 JM - added new URL

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by RLHecht** [comment]
There is now one central address for pension management centers, so this page is no longer needed and can be archived. We've already updated pages where we crosslink to it. We aren't sure where this should redirect to but wanted IA input. 

Also noting that this is listed in the left nav so we'd need to remove it there as well.

**Comment by jennymayoco** [comment]
Hi @jilladams @FranECross we've decided on the `/pension/how-to-apply/` as the destination URL so I've updated info in the ticket. The PMC page is being taken down and so there is only 1 location where folks can send their form by mail so we recommend redirecting folks to the how to apply page since there's a section for how to apply by mail. This is good to go. Thanks!

**Comment by jennymayoco** [comment]
Noting here that once the redirect is implemented, we can archive it and remove it from the left nav

cc @RLHecht

**Comment by randimays** [comment]
The code for the redirect is merged; it will deploy on Monday Aug 5 around 9am CST.

**Comment by randimays** [comment]
@RLHecht (or @jennymayoco?) this redirect is in place and is working correctly for me. Could you verify on your end and close the ticket if all is well?

**Comment by randimays** [comment]
@FranECross Forgot to add here that this was pulled in as stretch for sprint 9.

**Comment by jennymayoco** [comment]
Hi @randimays, thanks for the update! Confirming redirects looks good so closing this ticket
